### DISCUSSION TITLE
unable to render rich display in github problem solved

### DISCUSSION THREAD

**Discussion by ayrotideysarkar** [community/community#41020] [Actions]
### Select Topic Area

Bug

### Body

I am facing this issue can anybody help me with this?

**Comment by stevengj** [comment]
I'm seeing the same thing — about 1 week ago, github's rendering of Jupyter notebooks was broken.   Very simple notebooks still render, but lots of notebooks fail, e.g. ~~[this file now fails to render](https://github.com/mitmath/1806/blob/master/psets/pset3.ipynb)~~ [updated link](https://github.com/mitmath/1806/blob/fall22/psets/pset3.ipynb):

<img width="1140" alt="image" src="https://user-images.githubusercontent.com/2913679/206457501-192ef66e-9666-4d0e-a8dd-ba38a26f3bf2.png">

A partial workaround is to display Jupyter notebooks via [nbviewer.org](https://nbviewer.org/).   

nbviewer.org is great, but (a) it is an extra step to paste the github link into nbviewer.org, (b) it takes about a day for a new file to show up at nbviewer.org without giving a 404 for some reason, and (c) the nbviewer.org results are cached so it takes another day for them to update if you change the file on github.

**Reply by ayrotideysarkar** [reply]
@stevengj 
I searched every single site but didn't find the solution then I tried to change DNS and my problem solved. You can try this method also, I am mentioning the link below you can check the instructions
https://www.lifewire.com/how-to-change-dns-servers-in-windows-2626242

**Reply by stevengj** [reply]
I think you're talking about something completely different if changing DNS servers helped.

For example, I'm assuming [this github page](https://github.com/mitmath/1806/blob/master/psets/pset3.ipynb) still fails to render for you?

**Reply by ayrotideysarkar** [reply]
No I checked but https://github.com/mitmath/1806/blob/master/psets/pset3.ipynb
I am not able to see anything 🙂

**Reply by joshiayush** [reply]
@stevengj I don't understand why would I use nbviewer when I want to render my IPython Notebooks in GitHub?

Look at this file https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/classification.ipynb, are they using nbviewer there? Why GitHub is rendering rich text for tensorflow and not for me like this one https://github.com/joshiayush/ai/blob/master/ai/algos/correlation/pearson_correlation/Pearson_Correlation_Coefficient.ipynb. Now, after spending two years with GitHub do I have to find an alternative of it where I can transfer all my projects and say fucking good bye to GitHub?

Neither it renders rich text for me nor it lists code files under the language section in my repo, see https://github.com/joshiayush/ai/search?l=jupyter-notebook. Now, I'm getting mad on GitHub because since I created that repo it is consistent with its fucking behaviour.

**Comment by ayrotideysarkar** [comment]
Yup... But when I connect to vpn then I was able to see any images file so
I thought the issue I was facing based on my location so I changed DNS and
it worked

**Reply by riya-sharma6** [reply]
Guys changing DNS didn't help. Any solutions???

**Reply by 0xSunill** [reply]
change dns its working
watch this video, do the first solution it's working
https://youtu.be/1MMnUBsUczY

**Reply by ChakrahariHarshaVardhanRaju** [reply]
Thank You after changing the dns server its working

**Reply by kushagraahire** [reply]
> change dns its working watch this video, do the first solution it's working https://youtu.be/1MMnUBsUczY

Thanks it worked

**Reply by DevYashFlutter** [reply]
Thanks, guys, now it's working just after changing the DNS.

**Reply by RuizeZhou** [reply]
it works!thank you

**Reply by pritamlaskar** [reply]
Thanks! The first solution from the video truly works!

**Comment by 0xSunill** [comment]
> ### Body
> I am facing this issue can anybody help me with this?

change dns it's working
watch this video, do the first solution it's working
https://youtu.be/1MMnUBsUczY

**Comment by RukashDG** [comment]
I fixed this issue check my page,  change DNS its working.
https://github.com/RukashDG

**Reply by ayrotideysarkar** [reply]
Great 😊

**Comment by ayrotideysarkar** [comment]
Guys to solve this issue everyone needs to change their DNS

**Reply by Sawan7588** [reply]
Change DNS fixed my issue Thank you!!!

**Comment by ClementWalter** [comment]
I have started to have the same issue I don't know why I should change DNS settings, is there any comments from @github-staff about this ?

see for example https://github.com/daira/halographs/blob/master/halographs.pdf pdf not rendered

**Comment by stevengj** [comment]
I think the DNS thing is a red herring — even @ayrotideysarkar still can't view the ~~[link I posted above](https://github.com/mitmath/1806/blob/master/psets/pset3.ipynb)~~ [updated link](https://github.com/mitmath/1806/blob/fall22/psets/pset3.ipynb) (which worked fine until 2 weeks ago).

I'm guessing there is some timeout in github's display rendering that was recently introduced.  If you have a file that is sufficiently complicated, it times out and fails to render.   If you have a file right on the borderline, then it sometimes displays if you are lucky, and @ayrotideysarkar's change of DNS just happened to coincide with hitting that boundary for one of their files.

(@ClementWalter, [your link](https://github.com/mitmath/1806/blob/master/psets/pset3.ipynb) renders fine for me though.)

**Reply by ClementWalter** [reply]
I realized that quite a lot of links were not rendered yesterday at the time of writing, but then magically appeared again

**Reply by ayrotideysarkar** [reply]
@stevengj is right. I am still not able to view that link's content. we should seek GitHub stuff's help.

**Reply by MaAleem08** [reply]
I got the same problem ' unable to render rich dispay'
Then I tried the solution changing DNS. It worked. But  again the same problem is displayed. I checked the DNS, its same what I changed to 8:8:8:8 and 8:8:4:4.

now iam in real trouble. Did anyone find a solution to this?

**Reply by MaAleem08** [reply]
> @stevengj is right. I am still not able to view that link's content. we should seek GitHub stuff's help.
I got the same problem ' unable to render rich dispay'
Then I tried the solution changing DNS. It worked. But again the same problem is displayed. I checked the DNS, its same what I changed to 8:8:8:8 and 8:8:4:4.

now iam in real trouble. Did anyone find a solution to this?

**Comment by stevengj** [comment]
The problem seems to be fixed for me: the [page I linked above](https://github.com/mitmath/1806/blob/fall22/psets/pset3.ipynb) (with an updated link since the repo has changed) works fine for me now.

**Reply by tomaszli** [reply]
> The problem seems to be fixed for me: the [page I linked above](https://github.com/mitmath/1806/blob/fall22/psets/pset3.ipynb) (with an updated link since the repo has changed) works fine for me now.

The page doesn't render the image, again. Same as diagrams in my personal "Gists". It didn't worked yesterday, and looks like it won't just "fix itself".
